fvm/environment/program_recovery.go

```diff
@@ -201,6 +201,10 @@ func RecoveredNonFungibleTokenCode(nonFungibleTokenAddress common.Address, contr
                       %[2]s.recoveryPanic("Collection.deposit")
                   }
 
+				  access(all) view fun getIDs(): [UInt64] {
```

Comment:
the formatting is a bit weird

Reply:
I think I was able to fix the formatting in my latest commit